ci_helpers/helpers.sh

```diff
@@ -7,8 +7,17 @@ enable_moto_server(){
   moto_server -p5000 2>/dev/null&
 }
 
+create_ftp_server(){
+  docker run -d -p 21:21 -p 21000-21010:21000-21010 -e USERS="user|123|/home/user/dir" -e ADDRESS=localhost delfer/alpine-ftp-server
```

Comment:
Do we really need docker for something like this? We can use a more lightweight solution here: https://pypi.org/project/python-ftp-server/

Reply:
That is interesting. I have looked into the python-ftp-server. An interesting difference between this server and the docker image used above is that the python ftp server creates an FTP server with all the files within the smart_open directory copied to it (which could lead to unnecessary overhead), whereas the docker image creates an empty ftp server. 

In addition, when writing to the FTP server created by the docker image, the original smart_open directory is not affected. However, with the python-ftp-server, the written files will also be written to the smart_open directory during ci. 

Do you think this behavior be problematic? If so, I can also look at other lightweight ftp servers if we want to avoid using docker at all costs. If not, I will utilize this lighter-weight ftp server.

Reply:
Also, in terms of another potential local ftp server that could be created, I found [this](https://github.com/nixawk/pentest-wiki/blob/master/System-Services/services/twisted-ftp.md).